docs/compass/04-01-compass.md

```diff
@@ -7,10 +7,24 @@ To enable Compass in Kyma, follow the instructions for the [custom component ins
 
 ## Default Kyma installation
 
-This is a single-tenant mode which provides the complete cluster Kyma installation with all components, including Compass and Runtime Agent. In this mode, Runtime Agent is already connected to Compass and they both work in a single-tenant mode as well. Using this mode, you can register external Applications in Kyma. To enable it, follow the cluster Kyma installation and use the [`installer-cr-cluster-with-compass.yaml.tpl`](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-cr-cluster-with-compass.yaml.tpl) configuration file.
-
 ![Kyma mode1](./assets/kyma-mode1.svg)
 
+This is a single-tenant mode which provides the complete cluster Kyma installation with all components, including Compass and Runtime Agent. In this mode, Runtime Agent is already connected to Compass and they both work in a single-tenant mode as well. Using this mode, you can register external Applications in Kyma. To enable it, follow the cluster Kyma installation using the [`installer-cr-cluster-with-compass.yaml.tpl`](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-cr-cluster-with-compass.yaml.tpl) configuration file, and apply the following ConfigMap which enables the [API Packages](https://github.com/kyma-incubator/compass/blob/master/docs/compass/03-packages-api.md) functionality:
```

Comment:
maybe we should highlight that this ConfigMap should be applied before the installation will start :)